lib/content-services/src/lib/tag/tag-node-list.component.ts

```diff
@@ -42,7 +42,20 @@ export class TagNodeListComponent implements OnChanges, OnDestroy, OnInit {
     @Input()
     showDelete = true;
 
-    tagsEntries: TagEntry[];
+    /** Should limit number of tags displayed */
+    @Input()
+    limitTagsDisplayed = false;
+
+    @ViewChild('nodeListContainer')
+    containerView: ElementRef;
+
+    @ViewChildren(MatChip)
+    tagChips: QueryList<MatChip>;
+
+    tagsEntries: TagEntry[] = [];
+    calculationsDone = false;
+    columnFlexDirection = false;
```

Comment:
same for this variable, data type not present.

Reply:
See above comment from Denys